Source/Engine/UI/GUI/Panels/TilesPanel.cs

```diff
@@ -83,15 +111,15 @@ protected override void PerformLayoutBeforeChildren()
 
                 c.Bounds = new Rectangle(x, y, itemsWidth, itemsHeight);
```

Comment:
Shouldn't the x,y for Bounds rectangle be offset by Time Margin Left,Right?

Reply:
You're right, just pushed another commit with that line fixed.